### ISSUE TITLE
Feature request: Manually arrange home menu

### ISSUE THREAD

**Issue by ZubaZ21** [kaaholst/android-squeezer#786]
As a user I would like to be able to change the order of menu options (like putting the favorites at the bottom or power on/off at the top.

Long-press and drag would be natural but would impact the archive function.

A control in Settings to drag and drop would also be natural.

**Comment by LindyBalboa** [comment]
I too wish this were an option! I often use Spotify (Spotty) and would like to open that directly from the home screen or even set it as default.

**Comment by kaaholst** [comment]
> I too wish this were an option! I often use Spotify (Spotty) and would like to open that directly from the home screen or even set it as default.

@LindyBalboa If enter My Apps and long press Spotty, should add it to the home screen. Doesn't that work, or did you mean something else?

**Comment by KleinTommy** [comment]
Yes, being able to rearrange items in the home menu would be nice.
For everyday use I really only need "My Music" and "Favourites", so I would like to be able to hide the other items.
